packages/fs-detectors/src/detect-framework.ts

```diff
@@ -135,6 +135,30 @@ async function matches(
   };
 }
 
+function removeSupercededFramework(
```

Comment:
Spelling?

```suggestion
function removeSupersededFramework(
```

Reply:
@cb1kenobi I don't think spelling is worth blocking the PR. I do think it should have been fixed, but could have been a follow-up PR.